### ISSUE TITLE
Support for using existing Secret for pulling image from Private Registry

### ISSUE THREAD

**Issue by iamhoges** [CrowdStrike/falcon-helm#132]
I am trying to play around with `values.yaml` file in the `image.pullSecrets` section to be able to use an existing kubernetes secret in the cluster. 

I ended up updating the manifest file to get the secret name from the values.yaml file, for example:
```
      imagePullSecrets:
      {{- if .Values.container.image.pullSecrets }}
        - name: {{ .Values.container.image.pullSecrets.name }}
      {{- end }}
```
values.yaml:
```
image:
    repository: my-private-registry/crowdstrike/falcon-sensor
    pullPolicy: IfNotPresent
    pullSecrets:
      name: my-private-registry-creds
```

**Comment by isimluk** [comment]
Hello,

I have come across this issue and I wonder whether just supplying 
```
--set serviceAccount.name="default"
```
to the installer solves the issue.

The reasoning behind this: I imagine people may want to supply their own pullSecretName when the pullSecret gets created in their kubernetes whenever the namespace gets created. In that case usually the said pullSecret is usually accessible using the default service account.

Thoughts?

**Comment by falcon-pioupiou** [comment]
This issue was solved with this PR : #151 - you can pass a pullsecret name for Falcon Container here: https://github.com/CrowdStrike/falcon-helm/blob/main/helm-charts/falcon-sensor/values.yaml#L139 and here for Falcon Sensor as DaemonSet : https://github.com/CrowdStrike/falcon-helm/blob/main/helm-charts/falcon-sensor/values.yaml#L51
